polylogue/local_sync.py

```
@@ -127,9 +139,12 @@ def _sync_sessions(
 
     diff_total = 0
     if isinstance(sessions, (list, tuple)):
-        iterable: Iterable[Path] = sorted(sessions, key=path_order_key)
+        iterable: Iterable[Path] = sorted((Path(p) for p in sessions), key=path_order_key)
     else:
-        iterable = sessions
+        iterable = sorted(list(sessions), key=path_order_key)
```

Ruff (0.15.9)

[warning] Unnecessary `list()` call within `sorted()`

Remove the inner `list()` call

(C414)